optimum/exporters/neuron/convert.py

```diff
@@ -458,27 +458,32 @@ def export_neuronx(
 
 
 def add_stable_diffusion_compiler_args(config, compiler_args):
-    if hasattr(config._config, "_name_or_path"):
-        sd_components = ["text_encoder", "vae", "vae_encoder", "vae_decoder"]
-        if any(component in config._config._name_or_path.lower() for component in sd_components):
+    identifier = (
+        getattr(config._config, "_name_or_path", "").lower() + " " + getattr(config._config, "_class_name", "").lower()
+    )
+    sd_components = ["text_encoder", "vae", "vae_encoder", "vae_decoder"]
+    if any(component in identifier for component in sd_components):
+        compiler_args.extend(["--enable-fast-loading-neuron-binaries"])
```

Comment:
I think you can just use `.append("--enable-fast-loading-neuron-binaries")` for adding a single item at the end of the list, no big deal

Reply:
I will use append, I guess I wanted to extend with a list before then gradually realise there are some flags only applicable for some models 😄